WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -26,6 +26,12 @@ public class SimulationApiModel
         [JsonProperty(PropertyName = "Id")]
         public string Id { get; set; }
 
+        [JsonProperty(PropertyName = "Name")]
+        public string Name { get; set; }
+
+        [JsonProperty(PropertyName = "Desc", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
I think you can drop the NullValueHandling, and let the API returng a null field. That will help a user willing to edit the description, changing it from null to something else